src/transformers/expect-js.test.js

```diff
@@ -0,0 +1,291 @@
+/* eslint-env jest */
+import chalk from 'chalk';
+import { wrapPlugin } from '../utils/test-helpers';
+import plugin from './expect-js';
+
+chalk.enabled = false;
+const wrappedPlugin = wrapPlugin(plugin);
+
+function testChanged(msg, source, expectedOutput, options = {}) {
+    test(msg, () => {
+        const result = wrappedPlugin(source, options);
+        expect(result).toBe(expectedOutput);
+
+        // Running it twice should yield same result
+        expect(wrappedPlugin(result, options)).toBe(result);
+    });
+}
+
+testChanged(
+    'does not touch code without expect require/import',
+    `
+    const test = require("testlib");
+    test(t => {
+      expect(stuff).toExist();
+    })
+    `,
+    `
+    const test = require("testlib");
+    test(t => {
+      expect(stuff).toExist();
+    })
+    `
+);
+
+testChanged(
+    'maps expect matchers',
+    `
+    import expect from 'expect.js';
+
+    test(() => {
+      expect(stuff).to.be.ok();
+      expect(stuff).to.not.be.ok();
+      expect(stuff).to.be.empty();
+
+      expect(stuff).to.be('message');
+      expect(stuff).to.equal('message');
+
+      expect(stuff).to.eql('message');
+
+      expect(stuff).to.contain('message');
+
+      expect(stuff).to.have.length(1);
+
+      expect(stuff).to.have.property('message');
+      expect(stuff).to.have.key('message');
+
+      expect(stuff).to.match('message');
+    });
+    `,
+    `
+    test(() => {
+      expect(stuff).toBeTruthy();
+      expect(stuff).toBeFalsy();
+      expect(stuff).toBeFalsy();
```

Comment:
> empty: asserts that an array is empty or not

So `expect(stuff).to.be.empty();` should be translated to `expect(stuff).toHaveLength(0)`.